test_my_gspread.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+import gspread
+import pandas as pd
+from pandas import json_normalize
+from dotenv import load_dotenv
+import os
+import requests
+import json
+import datetime as dt
+
+from my_gspread import get_or_create_worksheet
+from my_gspread import pass_json_into_df
+
+load_dotenv()
+gspread_key = os.getenv('Project_key')
+gc = gspread.service_account(gspread_key)
```

Comment:
I think I'd prefer, you don't do this. When writing tests, a good practice is to avoid anything that will call APIs or connect to an external network outside your environment. So the way to go is to use mocks, If you try to run a test that connects over an API you're not unit-testing your code because you're now testing the code of the API inside your own code, the goal of this project is unit testing not integration testing